slsim/Util/param_util.py

```diff
@@ -85,3 +86,60 @@ def magnitude_to_amplitude(magnitude, mag_zero_point):
     delta_m = magnitude - mag_zero_point
     counts = 10 ** (-delta_m / 2.5)
     return counts
+
+
+def images_to_pixels(image_series):
+    """Converts a series of image snapshots into a list of pixel snapshots.
+
+    :param image_series: Series of images to convert
+    :return: List of pixel snapshots
+    """
+    initial_shape = np.shape(image_series)
+    number_of_pixels = initial_shape[1] * initial_shape[2]
+    return np.reshape(image_series, (initial_shape[0], number_of_pixels))
+
+
+def pixels_to_images(pixels, original_shape):
+    """Converts a string of pixel snapshots back into a series of image snapshots. This
+    is the inverse of images_to_pixels.
+
+    :param pixels: Series of pixel snapshots to arrange back into the original image
+        shape
+    :param original_shape: The original shape of the image snapshots
```

Comment:
I guess this is a tuple? Perhaps specify

Reply:
added clarification